salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -48,20 +48,23 @@ Create containerd service drop-in:
     - makedirs: true
     - dir_mode: 0755
     - context:
-      environment:
-      {%- if proxies %}
-        {%- set no_proxy = ["localhost", "127.0.0.1"] + networks.values() %}
-        {%- if proxies.no_proxy | default %}
-          {%- do no_proxy.extend(proxies.no_proxy) %}
+        args:
```

Comment:
Minor: I would have named this variable `containerd_options` or `containerd_args`,  as `args` is not crystal clear.

Reply:
Done